api/Headers.json

```diff
@@ -0,0 +1,1106 @@
+            },
+            "chrome_android": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "edge": {
+              "version_added": "16",
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "44",
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "firefox_android": {
+              "version_added": "44",
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "opera_android": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "lexicographical_sorting": {
+        "__compat": {
+          "description": "Lexicographical sorting, and values from duplicate header names combined when iterated.",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "44"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "24"
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          }
+        }
+      },
+      "getall_removed": {
```

Comment:
Remove the whole entry for "getall_removed". This is redundant with the info in `getAll`.

Reply:
👍